modules/security-context-constraints-psa-about.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * authentication/understanding-and-managing-pod-security-admission.adoc
+// * operators/operator_sdk/osdk-complying-with-psa.adoc
+
+:_content-type: CONCEPT
+[id="security-context-constraints-psa-about_{context}"]
+= About pod security admission
+
+{product-title} includes link:https://kubernetes.io/docs/concepts/security/pod-security-admission[Kubernetes pod security admission]. Pods that do not comply with the pod security admission defined globally or at the namespace level are not admitted to the cluster and cannot run.
+
+Globally, the `privileged` profile is enforced, and the `restricted` profile is used for warnings and audits.
+
+You can also configure the pod security admission settings at the namespace level.
+
+[id="psa-modes_{context}"]
+== Pod security admission modes
+
+You can configure the following pod security admission modes for a namespace:
+
+.Pod security admission modes
+[cols="1,2,3a",options="header"]
+|===
+|Mode
+|Label
+|Description
+
+|`enforce`
+|`pod-security.kubernetes.io/enforce`
+|Rejects a pod from admission if it does not comply with the set profile.
```

Comment:
```suggestion
|Rejects a pod from admission if it does not comply with the set profile
```
I don't think the descriptions in this table need to end with periods because none of them are complete sentences. Reference: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-periods#lists